### PR TITLE
Add argMin and argMax operators

### PR DESCRIPTION
**Pull request by xhcao**
@qjia7 @axinging @haoyunfeix    PTAL

### CODE REVIEW

File: src/setup_test.ts [hunk 1]
```diff
@@ -76,8 +77,10 @@ const EXCLUDE_LIST: string[] = [
   'concat tensors with 0 in their shape',                // Timeout.
   'argmax test-webgpu {} accepts tensor with bool',      // Actual != Expected.
   'encodeWeights',                                       // Bool tensors fails.
-  'deprecation warnings',  // tf.disableDeprecationWarnings.
-  'oneHot',                // Not yet implemented
+  'deprecation warnings',             // tf.disableDeprecationWarnings.
+  'oneHot',                           // Not yet implemented.
+  'accepts tensor with bool values',  // Error happens in ANGLE.
```
Comment (qjia7) [inline src/setup_test.ts]: Please file an angle bug to track it. And past the link here.

File: src/backend_webgl2compute.ts [hunk 2]
```diff
@@ -292,6 +293,21 @@ export class WebGL2ComputeBackend extends KernelBackend {
     return this.compileAndRun(program, [x]);
   }
 
+  private argMinMaxReduce(x: Tensor, axis: number, reduceType: 'min'|'max'):
+    Tensor {
+  const program = new ArgMinMaxProgram(x.shape, axis, reduceType);
```
Comment (qjia7) [inline src/backend_webgl2compute.ts]: Indent. Please modify all related places.

File: src/setup_test.ts (hunk 1 above)
Comment (xhcao) [reply src/setup_test.ts]: This root cause is still in tfjs-compute, currently, bool type in tensor is not yet implemented.

**Comment by xhcao** [comment]
Please take a look again.

**Comment by xhcao** [comment]
Had changed uint format for scalar value.

**Comment by qjia7** [comment]
LGTM. Thanks.

**Pull request merged**